src/components/App.js

```diff
@@ -1,28 +1,39 @@
-import { Layout } from 'antd';
+import { Layout, notification } from 'antd';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { BrowserRouter, Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 
-import Error from './Error';
-import { getError } from 'console/state/auth/selectors';
-
 import AuthButton from 'console/components/auth/AuthButton';
 import NavBar from 'console/components/navigation/NavBar';
 import QueryActions from 'console/components/data/QueryActions';
 import QueryAuth0 from 'console/components/data/QueryAuth0';
 import AppRouter from 'console/components/AppRouter';
 import CircleLogo from 'console/components/svg/CircleLogo';
+import { getError } from 'console/state/auth/selectors';
+
 const { Header } = Layout;
 
 @connect((state, props) => ({
-  error: getError(state),
+  authError: getError(state),
 }))
 export default class App extends React.Component {
   static propTypes = {
-    error: PropTypes.object,
+    authError: PropTypes.object,
   };
 
+  componentDidUpdate() {
+    const { authError } = this.props;
+
+    if (authError) {
+      notification['error']({
+        message: 'Authentication Error',
+        description: `${authError.get('error')}: ${authError.get('errorDescription')}`,
```

Comment:
Somewhere near here it would be really useful to code document that this `authError` object is the one that comes from `auth0-js`.